### PR TITLE
show  alpha value of gradients dynamically

### PR DESCRIPTION
**Pull request by BerndN**

### CODE REVIEW

File: Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript [hunk 1]
```diff
@@ -87,6 +87,7 @@ on updateColorValue pColor
    lock screen
    lock messages
    set the swatchColor of widget "color" of me to pColor
+   put pColor into field "value" of me
```
Comment (montegoulding) [inline Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript]: @BerndN This patch effectively makes `updateColorValue` and `updateColorDisplay` exactly the same. I have a tendency to think if there's two such similar implementations there must be a reason for the difference. I'm sure @livecodeali will know!

Comment (BerndN) [inline Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript]: @montegoulding updateColorDisplay is not called when changing the alpha value by slider but updateColorValue is. So I went with the conservative approach just adding the fix to updateColorValue without fiddling with messaging.
As you say, @livecodeali  will know.

Comment (livecodeali) [inline Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript]: Unfortunately I'm not quite sure why there are two such similar implementations! 

Originally I was using separate implementations for updating the visual feedback of the chosen color value and updating the stored editor value - it looks like in the course of fixing things these two handlers have become effectively identical, so I think there is no problem with merging them into one handler.

Comment (BerndN) [inline Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript]: @livecodeali 
I updated the pull request and propose now to take handler "updateColorValue" out and change in 
handler "updateSubeditorValue" to "updateColorDisplay"
As far as I can tell it works and has no untoward effects

**Comment by livecodeali** [comment]
@BerndN Hi Bernd, your bugfix note's filename needs to be `bugfix-17759.md` and the description in it needs to start with a `#`.

**Comment by BerndN** [comment]
@livecodeali Done

**Comment by livecodeali** [comment]
@BerndN Thanks!
@livecode-vulcan review ok b45c5b8

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok b45c5b80d11e617b60f65ddd8ebe2205462fc8f6

**Pull request merged**